crud/result.go

```diff
@@ -75,10 +75,13 @@ func (r *Result) DecodeMsgpack(d *msgpack.Decoder) error {
 		return err
 	}
 
-	if arrLen < 2 {
+	if arrLen == 0 {
 		return fmt.Errorf("array len doesn't match: %d", arrLen)
```

Comment:
or something like this..
```suggestion
		return fmt.Errorf("unexpected empty response array")
```

Reply:
Updated, added missing zero length check to bool response converter